common/djangoapps/django_sudo/utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+sudo.utils
+"""
+import django.contrib.sessions.middleware
+import sudo.middleware
+
+
+def region_name(region):
```

Comment:
@cpennington No, it is for edx specific ```course_id``` region to make it valid for cookie. we are just removing these three character's ```[':', '/', '+']``` from ```course_id``` in this function.